src/main/kotlin/se/uulm/snowballr/backend/repository/association/ProjectPaperTableRepo.kt

```diff
@@ -129,6 +132,21 @@ interface IProjectPaperTableRepo {
      */
     suspend fun getProjectProgress(projectId: UUID): Float
 
+    /**
+     * Retrieves a list of subsequent project papers from a given starting point defined by `localPaperId` and `stage`.
+     *
+     * Subsequent project papers are defined as project papers
+     *  - within the same project
+     *  - with a greater `localPaperId` than the given project paper and
+     *  - are part of the given `stage` or a subsequent `stage`.
```

Comment:
I think this says that the `localPaperId` always has to be greater, which is not the case. Sorry for being so meticulous, but I think it's important that this is clear.

```suggestion
     * Subsequent project papers are defined as project papers within the same project that are either:
     * - in the same `stage` but have a greater `localPaperId`, or
     * - in a subsequent `stage` regardless of their `localPaperId`.
```